src/main/java/eightbit/command/FindCommand.java

```diff
@@ -0,0 +1,58 @@
+package eightbit.command;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+import eightbit.task.Task;
+import eightbit.util.Storage;
+import eightbit.util.TaskList;
+
+/**
+ * Represents a command for finding tasks based on some keywords.
+ */
+public class FindCommand extends Command {
+
+    private final String keyword;
+
+    /**
+     * Constructs a command for finding tasks.
+     *
+     * @param keyword Search keyword.
+     */
+    public FindCommand(String keyword) {
+        this.keyword = keyword.toLowerCase(Locale.ROOT); // allows case-insensitive search
+    }
+
+    /**
+     * Retrieves the tasks containing the search keyword and prints them.
+     *
+     * @param taskList User's list of tasks.
+     * @param storage  Storage responsible for reading/writing the file.
+     * @return The response after executing the command.
+     */
+    @Override
+    public String execute(TaskList taskList, Storage storage) {
+        // Get tasks containing the keyword
+        ArrayList<Task> results = new ArrayList<>();
+        for (int i = 0; i < taskList.size(); i++) {
+            Task task = taskList.get(i);
+            if (task.getDescription().toLowerCase(Locale.ROOT).contains(keyword)) { // case-insensitive search
```

Comment:
The comment should be indented relative to its position in the code, in this case in line with the if (task.getDescription()...) statement.